webapp/components/interop.js

```diff
@@ -917,18 +894,30 @@ class Interop2022Summary extends PolymerElement {
     };
   }
 
+  static get observers() {
+    return [
+      'updateYearInfo(year)'
+    ];
+  }
+
   _stableChanged() {
     this.updateSummaryScores();
   }
 
+  updateYearInfo(year) {
+    const yearInfo = PARAMS_BY_YEAR[year];
```

Comment:
I am not the biggest fan of using `PARAMS_BY_YEAR` directly within this file. It almost feels like it defeats the purpose of `InteropDataManager`, which "encapsulates the loading of the CSV data that backs
// both the summary scores and graphs shown on the Interop dashboard". In other words, the `view` class directly interacts with the `model`, in this case the JSON data, and it violates the MVC principle. Not sure if it is a stretch but perhaps @jcscottiii can chime in here?

Reply:
`InteropDataManager` loads and formats the CSV data for stable and experimental runs to use in percentage score views and the graph, and was not originally written to hold the data stored in the json file (which previously was hardcoded throughout the file). However, I'm not certain I'm handling this exactly as I should and I'm up for refactoring it - did you have a different approach in mind?

Reply:
Per our in-person discussion - we can move all the `updateYearInfo` in multiple instances into data manager and passing down dataManager to make it cleaner.